README.md

```diff
@@ -103,6 +103,7 @@ Digital Forensics and Incident Response (DFIR) teams are groups of people in an
 * [CyLR](https://github.com/orlikoski/CyLR) - The CyLR tool collects forensic artifacts from hosts with NTFS file systems quickly, securely and minimizes impact to the host.
 * [Forensic Artifacts](https://github.com/ForensicArtifacts/artifacts) - Digital Forensics Artifact Repository
 * [ir-rescue](https://github.com/diogo-fernan/ir-rescue) - Windows Batch script and a Unix Bash script to comprehensively collect host forensic data during incident response.
+* [kube-forensics](https://github.com/keikoproj/kube-forensics) - Kubernetes IR tool that allows an admin to dump the current state of a running pod and all its containers for off-line forensic analysis.
```

Comment:
@amir9339 is this project maintained ? seems no activity on it in past 2 years

Reply:
Hi @meirwah. No, as I know it's not maintained anymore.
I added this project because it's one of the only open-source Kubernetes forensics tools that I know about. 
I thought it will be useful to add a tool that collects Kubernetes artifacts to this list as first aid for investigating Kubernetes.

Reply:
I think we should try to add to list project which are actively maintained , let's remove it otherwise